llvm/include/llvm/TableGen/DirectiveEmitter.h

```diff
@@ -15,7 +15,8 @@ namespace llvm {
 // DirectiveBase.td and provides helper methods for accessing it.
 class DirectiveLanguage {
 public:
-  explicit DirectiveLanguage(llvm::RecordKeeper &Records) : Records(Records) {
+  explicit DirectiveLanguage(const llvm::RecordKeeper &Records)
```

Comment:
Is llvm:: needed here?

Reply:
No, but I refrained from changing it. Apart from this, there are some other NFC cleanups that can be done in this file. For example, using {} instead of LLVM's recommended () for constructors. and other llvm:: that's not needed. How about I clean that up as a separate NFC?

Reply:
> How about I clean that up as a separate NFC?

Sure